src/com/epam/javatrainings/classdecomposition/pizzeria_palmetto/Console.java

```diff
@@ -0,0 +1,45 @@
+package com.epam.javatrainings.classdecomposition.pizzeria_palmetto;
+
+import com.epam.javatrainings.classdecomposition.pizzeria_palmetto.model.Customer;
+import com.epam.javatrainings.classdecomposition.pizzeria_palmetto.model.Pizza;
+import com.epam.javatrainings.classdecomposition.pizzeria_palmetto.model.PlacingAnOrder;
+import com.epam.javatrainings.classdecomposition.pizzeria_palmetto.services.PlacingAnOrderService;
+import com.epam.javatrainings.classdecomposition.pizzeria_palmetto.services.PizzaService;
+import static com.epam.javatrainings.classdecomposition.pizzeria_palmetto.services.Constants.*;
+
+public class Console {
+
+  public static void main(String[] args) {
+
+    // Created customer1 and customer2 where they give  name and id number
+    Customer customer1 = new Customer(NAME_CUSTOMER_1, CUSTOMER_NUMBER_1);
+    Customer customer2 = new Customer(NAME_CUSTOMER_2, CUSTOMER_NUMBER_2);
+    // Customer can create an orders
+    PlacingAnOrder placingAnOrder1 = new PlacingAnOrder(customer1);
+    PlacingAnOrder placingAnOrder2 = new PlacingAnOrder(customer2);
+    PlacingAnOrderService orderService1 = new PlacingAnOrderService(placingAnOrder1);
+    PlacingAnOrderService orderService2 = new PlacingAnOrderService(placingAnOrder2);
+    // Add  ingredient to pizza
+    Pizza pizza = orderService1.create(NAME_PIZZA_1, TYPE_PIZZA_1, QUANTITY_PIZZA_1);
+    PizzaService pizzaService = new PizzaService(pizza);
```

Comment:
bad idea to create a pizzaService for each new pizza, please redesign the service

Reply:
done